server/schedule/labeler/labeler.go

```diff
@@ -78,7 +130,11 @@ func (l *RegionLabeler) loadRules() error {
 
 func (l *RegionLabeler) buildRangeList() {
 	builder := rangelist.NewBuilder()
+	l.earlistExpireTime = unlimittedExpire
```

Comment:
Perhaps, we can use the first rule's expiration time as the earlistExpireTime then we can avoid defining `unlimittedExpire` as a global variable.

Reply:
Yes, we can use the first rule's expire-time instead of `unlimittedExpire` here, however, to totally void to use `unlimittedExpire`, there may be too many conditions we need to consider, for example, With the `Labels` is empty we need to check both the `Labels` empty thing and `earlistExpireTime` in GC.

Reply:
The purpose is trying to avoid using the init function.